### DISCUSSION TITLE
New Visual Studio solution file format slnx XML based

### DISCUSSION THREAD

**Discussion by martin-honnen** [ikvmnet/ikvm#513] [Ideas]
It seems Microsoft is experimenting with a new Visual Studio file format that is XML based, suffix is `slnx`. I haven't seen anything from Microsoft itself unfortunately documenting it, but there are some blog entries like https://schwabencode.com/blog/2024/04/10/welcome-new-visual-studio-slnx-solution-file#:~:text=Microsoft%20has%20now%20tackled%20the,activated%20manually%20in%20the%20options. that mention it.
Seems like a good option to make creating/editing IKVM.Net.Sdk solutions/projects easier.

**Comment by GeorgeS2019** [comment]
Thx for sharing

**Comment by GeorgeS2019** [comment]
@martin-honnen 
is there a syntax highlighter for java that could be ported to e.g. VS2022 extension when using ikvm to compile java codes?

**Comment by martin-honnen** [comment]
@GeorgeS2019 , I am sorry, I don't know whether there is a syntax highlighter for the purpose you are looking for.
